src/bot/commands/social/insult.ts

```diff
@@ -0,0 +1,127 @@
+import { Command, Input } from '@api';
+import { GuildMember} from 'discord.js';
+import * as FS from 'fs';
+
+const filepath = 'random/insults.txt';
+const insults = readPublicFile(filepath).split(/[\n]/);
+
+export class Insult extends Command {
+    constructor(){
+        super({
+            name: 'insult',
+            aliases: ['mean'],
+            description: 'Insult someone.',
+            arguments: [{
+                name: 'user',
+                description: 'The scumbag to insult!!!!',
+                constraint: 'mention',
+                required: true
+            },
+            {
+                name: 'new_insult',
+                description: 'the insult to add to the list',
```

Comment:
The string the use passes is never actually 'added' to any list. Consider rewording this description.